### PR TITLE
Vendor dashboard data cleanup km

### LINKED ISSUE #31: When a user clicks a filter dropdown option
User should be able to sort weddings by different dates:
- [ ] past
- [ ] present/today?
- [ ] future

### PR DESCRIPTION
**Pull request by Kristiannmiller**
# Pull Request

## What's this PR do? 

- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing
- [X] Styling
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description of changes?
- Adds functionality to capitalize name string before posting
- Formats dates to display as "MM/DD/YYYY" on VendorDashboard
- Installs dependencies for dayjs
- adds functionality for weddings to sort according to date on vendordashboard
- Adds sort functionality to dropdown menu on VendorDashboard
- Adjusts minor styling

## Any background context you want to provide? 
Will need to run npm install on your local after pulling down

## What are the relevant tickets? 
* Closes #31

### CODE REVIEW

File: src/components/AddWeddingForm/AddWeddingForm.tsx [hunk 1]
```diff
@@ -34,6 +36,12 @@ const AddWeddingForm: React.FC = () => {
     clearInputs();
   }
 
+  const capitalize = (s: string) => {
+    if (typeof s !== 'string') return ''
+    const capitalized = s.charAt(0).toUpperCase() + s.slice(1)
+    setName(capitalized)
+    }
+
```
Comment (ericberglund117) [inline src/components/AddWeddingForm/AddWeddingForm.tsx:44]: Nice call capitalizing the name!

File: src/components/VendorDashboard/VendorDashboard.tsx [hunk 2]
```diff
@@ -5,59 +5,130 @@ import { getWeddings } from '../../apiCalls'
...
+          setWeddingFilter(sortedFilter)
+        } else {
+          setError(true)
+          setErrorMessage('No Weddings Taking Place Today')
+        }
+        break;
+      default:
+        setWeddings(weddings)
+        setWeddingFilter([])
+        break;
```
Comment (ericberglund117) [inline src/components/VendorDashboard/VendorDashboard.tsx:76]: Interesting conditional here. Glad you got it figured out. This is a complicated thing to think/plan through

**Review by ericberglund117 — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -11,6 +11,7 @@
     "@types/react-dom": "^17.0.0",
     "@types/react-router-dom": "^5.1.7",
     "@types/styled-components": "^5.1.7",
+    "dayjs": "^1.10.3",
```
Comment (hannakim91) [inline package.json:14]: I've never used this library before - neat find!

File: src/components/VendorDashboard/VendorDashboard.tsx (hunk 2 above)
Comment (hannakim91) [reply src/components/VendorDashboard/VendorDashboard.tsx:76]: Seems like a good place to use switch statements!

**Review by hannakim91 — APPROVED** [review]
Thanks for working on this feature!

**Pull request merged**